app/oauth.py

```diff
@@ -0,0 +1,81 @@
+import json
+import os
+import re
+
+import requests
+from django.contrib.auth.models import User as DefaultUser
+from dotenv import load_dotenv
+from oauth2_provider.models import AccessToken
```

Comment:
Please rearrange imports following the [PEP 8 guideline ](https://www.python.org/dev/peps/pep-0008/#imports)

```
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```